### PR TITLE
Add support for static string array

### LINKED ISSUE #211: An Issue for stringArray in smart contract when compile
Refer to https://github.com/neo-project/neo-devpack-dotnet/pull/187#discussion_r394673272.
`static readonly string[] SupportedStandards = new string[] { "NEP-5", "NEP-10" };` 
This kind of style can cause compilation error.
```
2>Find entrypoint:System.Object Template.NEP5.CSharp.NEP5::Main(System.String,System.Object[])
2>EXEC : Convert error : System.Exception: sth error
2>   at Neo.Compiler.MSIL.ModuleConverter.LinkCode(String main) in D:\Master-smoking\2020-03-20\neo-devpack-dotnet\src\Neo.Compiler.MSIL\MSIL\Converter.cs:line 359
2>   at Neo.Compiler.MSIL.ModuleConverter.Convert(ILModule _in, ConvOption option) in D:\Master-smoking\2020-03-20\neo-devpack-dotnet\src\Neo.Compiler.MSIL\MSIL\Converter.cs:line 200
2>   at Neo.Compiler.Program.Compile(Options options) in D:\Master-smoking\2020-03-20\neo-devpack-dotnet\src\Neo.Compiler.MSIL\Program.cs:line 159
2>D:\Master-smoking\2020-03-20\neo-devpack-dotnet\templates\Template.NEP5.CSharp\Template.NEP5.CSharp.csproj(12,5): error MSB3073: The command "neon -f "D:\Master-smoking\2020-03-20\neo-devpack-dotnet\templates\Template.NEP5.CSharp\bin\Debug\netstandard2.1\Template.NEP5.CSharp.dll" -o" exited with code -1.
2>Done building project "Template.NEP5.CSharp.csproj" -- FAILED.
```

### PR DESCRIPTION
**Pull request by Tommo-L**
close #211

### CODE REVIEW

**Comment by Tommo-L** [comment]
@lightszero @shargon plz, have a review.

File: src/Neo.Compiler.MSIL/MSIL/CctorSubVM.cs [hunk 1]
```diff
@@ -32,7 +32,12 @@ public static object Dup(object src)
                 Boolean v = (Boolean)src;
                 return v;
             }
-            else
+            else if (src.GetType() == typeof(string[]))
```
Comment (shargon) [inline src/Neo.Compiler.MSIL/MSIL/CctorSubVM.cs]: Could you add ushort[], short, long...etc etc ?

or... `src.GetType().IsArray`?

Comment (shargon) [reply src/Neo.Compiler.MSIL/MSIL/CctorSubVM.cs]: Wy we need to cast the objects?

Comment (Tommo-L) [reply src/Neo.Compiler.MSIL/MSIL/CctorSubVM.cs]: hm.m.....

I think we don't need. @lightszero  What dou you think?

Comment (lightszero) [reply src/Neo.Compiler.MSIL/MSIL/CctorSubVM.cs]: we can and ushort[] in future if we needed and merge this first. nep5 template need this.

**Pull request merged**